src/ui/public/timepicker/pretty_duration.js

```diff
@@ -20,47 +20,67 @@
 import dateMath from '@elastic/datemath';
 import moment from 'moment';
 import { timeUnits } from './time_units';
+import { i18n } from '@kbn/i18n';
+
+const TIME_NOW = 'now';
 
 function cantLookup(timeFrom, timeTo, dateFormat) {
   const displayFrom = formatTimeString(timeFrom, dateFormat);
   const displayTo = formatTimeString(timeTo, dateFormat, true);
-  return `${displayFrom} to ${displayTo}`;
+  return i18n.translate('common.ui.timepicker.fullTimeRange', {
+    defaultMessage: '{displayFrom} to {displayTo}',
+    values: {
+      displayFrom,
+      displayTo
+    }
+  });
 }
 
 function formatTimeString(timeString, dateFormat, roundUp = false) {
   if (moment(timeString).isValid()) {
     return moment(timeString).format(dateFormat);
   } else {
-    if (timeString === 'now') {
-      return 'now';
+    if (timeString === TIME_NOW) {
+      return i18n.translate('common.ui.timepicker.timeNow', { defaultMessage: 'now' });
     } else {
       const tryParse = dateMath.parse(timeString, { roundUp: roundUp });
       return moment.isMoment(tryParse) ? '~ ' + tryParse.fromNow() : timeString;
     }
   }
 }
 
+function _getDateLookupKey(startDate, endDate) {
```

Comment:
We don't generally prefix things with an `_`. They are private simply by not being exported. I thought there was a linter rule about this, but maybe not?

Reply:
I actually followed the styleguide [js_style_guide.js](https://github.com/elastic/kibana/blob/d5b2c8eaf2d766d28f6a0a7546b93931cc22cb57/style_guides/js_style_guide.md).

I think we don't follow that, should we remove it?